Dockerfile

```diff
@@ -1,12 +1,9 @@
 FROM alpine:3.5
 
-RUN apk --no-cache add bash curl jq wget groff less python py-pip &&\
+RUN apk --no-cache add bash curl jq wget groff less python py-pip docker &&\
```

Comment:
Installing `docker` from the packages might introduce unexpected version change.  I would rather stay pulling a specific version directly from Docker.  See https://github.com/mbentley/docker-docker-builder/blob/master/Dockerfile#L3-L9 for an example.